### DISCUSSION TITLE
Randomized note position

### DISCUSSION THREAD

**Discussion by Nproductionss** [Leather128/LeathersFunkinEngine#235] [Ideas]
I think it is cool if there was a randomize note button in the chart editor that move all notes to a random section which would make it easier to turn 4k charts into multikey charts without having to move them yourself  or have it be an option in settings just like in  Zoro engine

**Comment by Nproductionss** [comment]
photo explanation since idk if I explained it well see I have this chart which i turned into 6k 
![image](https://user-images.githubusercontent.com/87342196/163686764-d40c5554-8df3-4f59-a495-649aeeaca843.png)
then if I pressed the randomize note button it would  basically do this automatically for the whole chart 
![image](https://user-images.githubusercontent.com/87342196/163686789-fe2489c4-a2ff-452a-8258-2eee3ce42280.png)
it takes hour to turn in a 4 key chart into for example here a 6k chart but with this button in chart editor it make it done in the click of a button

**Reply by JordanSantiagoYT** [reply]
seems like a good idea but the chart randomizer also has a chance to make a weird chart
